superset-frontend/src/SqlLab/components/QuerySearch.jsx

```diff
@@ -39,6 +39,15 @@ const propTypes = {
   displayLimit: PropTypes.number.isRequired,
 };
 
+const TableStyles = styled.div`
+  height: ${props => props.height};
+
+  .table > thead > tr > th {
+    border-bottom: 2px solid #cfd8dc;
+    background: #f5f5f5;
```

Comment:
```suggestion
    background: ${({ theme }) => theme.colors.grayscale.light4};
```
Trying to avoid hard-coded colors/fonts/etc and use the Theme wherever possible.